roles/st2chatops/tasks/main.yml

```diff
@@ -0,0 +1,67 @@
+---
+- name: Install latest st2chatops package
+  become: yes
+  package:
+    name: st2chatops
+    state: latest
+  when: st2chatops_version == "latest"
+  tags: [st2chatops, skip_ansible_lint]
+
+- name: Install pinned st2chatops package
+  become: yes
+  package:
+    name: st2chatops={{ st2chatops_version }}
+    state: present
+  when: st2chatops_version != "latest"
+  tags: [st2chatops, skip_ansible_lint]
+
+- name: Add st2_api_key in st2chatops.env
+  become: yes
+  replace:
+    dest: /opt/stackstorm/chatops/st2chatops.env
+    regexp: '(?<=ST2_API_KEY=)(.*)$'
+    replace: '{{ st2chatops_config.ST2_API_KEY }}'
+  when: st2chatops_config.ST2_API_KEY != "CHANGE-ME-PLEASE"
+  register: st2chatops_api_key
+  tags: st2chatops
+
+- name: Comment Username, Password and Auth URL, if API_KEY provided
+  become: yes
+  replace:
+    dest: /opt/stackstorm/chatops/st2chatops.env
+    regexp: '{{ item }}'
+    replace: '# \1'
+  with_items:
+    - '^(export ST2_AUTH_URL.*)'
+    - '^(export ST2_AUTH_USERNAME.*)'
+    - '^(export ST2_AUTH_PASSWORD.*)'
+  when: st2chatops_api_key.changed
+  tags: [st2chatops, skip_ansible_lint]
+
+- name: Uncomment Hubot Slack Adapter in st2chatops.env
+  become: yes
+  replace:
+    dest: /opt/stackstorm/chatops/st2chatops.env
+    regexp: '^# (export HUBOT_ADAPTER=slack)$'
+    replace: '\1'
+  when: st2chatops_api_key.changed and st2chatops_config.HUBOT_ADAPTER == "slack"
+  register: uncomment_slack_adapter
+  tags: [st2chatops, skip_ansible_lint]
+
+- name: Uncomment and replace Hubot Slack Token in st2chatops.env
+  become: yes
+  replace:
+    dest: /opt/stackstorm/chatops/st2chatops.env
+    regexp: '^# (export HUBOT_SLACK_TOKEN.).*'
```

Comment:
Just googled a bit more about other implementations and "what could be expected".

See this config examples, looks pretty reasonable:
Ansible example for Hubot: https://github.com/brianshumate/ansible-hubot
Chef example for Hubot: https://github.com/chef-cookbooks/hubot#attributes

I think we can do it in the same way: 
Make `{{ st2chatops_adapter }}` var required (since hubot won't work without adapter) and additional env vars via `{{ st2chatops_config }}` dict optional.

The problem with hardcoded vars is that apart of "adapter vars" there are also hubot plugin vars:
https://github.com/brianshumate/ansible-hubot/blob/5a1d679472ebfd12119cc9db0209a219560cc6ab/defaults/main.yml#L61